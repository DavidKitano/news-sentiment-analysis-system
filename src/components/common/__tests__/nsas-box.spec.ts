import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import NsasBox from '../nsas-box.vue'

describe('NsasBox', () => {
  it('should render the component', () => {
    const wrapper = mount(NsasBox)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
