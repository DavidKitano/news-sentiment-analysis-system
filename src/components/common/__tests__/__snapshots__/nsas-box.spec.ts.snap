// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NsasBox > should render the component 1`] = `
"<div class="nsas-box">
  <div class="el-card is-hover-shadow">
    <!--v-if-->
    <div class="el-card__body" style=""></div>
    <!--v-if-->
  </div>
</div>"
`;
